### PR TITLE
use new query utils for post_proofers directory

### PR DESCRIPTION
**Pull request by chrismiceli**
Will put in sandbox after my srchrep PR is merged or I figure out how to host 2 branches at the same time.

### CODE REVIEW

File: tools/post_proofers/postcomments.php [hunk 1]
```diff
@@ -18,11 +18,12 @@
   exit;
 }
 
-$qry =  mysqli_query(DPDatabase::get_connection(), sprintf("
-    UPDATE projects
-    SET postcomments = '%s'
-    WHERE projectid = '$projectid'
-", mysqli_real_escape_string(DPDatabase::get_connection(), $postcomments)));
+$sql = sprintf("
+  UPDATE projects
```
Comment (cpeel) [inline tools/post_proofers/postcomments.php:22]: Please indent these 4 spaces.

File: tools/post_proofers/ppv_report.php [hunk 2]
```diff
@@ -85,12 +85,13 @@
...
 // number of books post-processed by this PPer (including this one).
 $psd = get_project_status_descriptor('PPd');
-$result = mysqli_query(DPDatabase::get_connection(), "
+$sql = sprintf("
     SELECT COUNT(*) AS num_post_processed
     FROM projects
     WHERE $psd->state_selector
-      AND postproofer = '$project->postproofer'
-");
+    AND postproofer = LEFT('%s', 25)
```
Comment (cpeel) [inline tools/post_proofers/ppv_report.php:92]: Please keep this indented from the `WHERE`, ideally by 4 spaces.

Comment (cpeel) [reply tools/post_proofers/ppv_report.php:92]: And nice use of the `LEFT()`. 👍

File: tools/post_proofers/ppv_report.php [hunk 3]
```diff
@@ -106,28 +107,38 @@
 $pp_date = "";
 
 // earliest transition from PPV.avail to PPV.checked out
-$result = mysqli_query(DPDatabase::get_connection(), "SELECT timestamp FROM project_events
-    WHERE projectid = '$projectid'
+$sql = sprintf("SELECT timestamp FROM project_events
```
Comment (cpeel) [inline tools/post_proofers/ppv_report.php:110]: Move `SELECT` down to it's own line.

File: tools/post_proofers/ppv_report.php [hunk 4]
```diff
@@ -106,28 +107,38 @@
...
 $row = mysqli_fetch_assoc($result);
 mysqli_free_result($result);
 if ($row)
 {
     $earliest_in_ppv = $row["timestamp"];
 
     // latest transition from PP.checked out to PPV.avail
-    $result = mysqli_query(DPDatabase::get_connection(), "SELECT timestamp FROM project_events
-        WHERE projectid = '$projectid'
+    $sql = sprintf("SELECT timestamp FROM project_events
```
Comment (cpeel) [inline tools/post_proofers/ppv_report.php:128]: Ditto `SELECT` on its own row.

File: tools/post_proofers/ppv_report.php (hunk 2 above)
Comment (chrismiceli) [reply tools/post_proofers/ppv_report.php:92]: I see, I copy :)

**Comment by chrismiceli** [comment]
Updated sandbox as well https://www.pgdp.org/~chrismiceli/c/

**Pull request merged**